suse-module-tools.spec

```diff
@@ -192,44 +180,88 @@ test_allow_on_install()
 		return
 	fi
 }
-# upgrade from old locations
-if test -e %{_sysconfdir}/modprobe.d/unsupported-modules; then
-	mv -f %{_sysconfdir}/modprobe.d/unsupported-modules \
-		%{_sysconfdir}/modprobe.d/10-unsupported-modules.conf
-fi
 test_allow_on_install "$@"
 if test -n "$allow" -a "$allow" != "$allowed"; then
 	sed -ri 's/^( *allow_unsupported_modules *) [01]/\1 '"$allow"'/' \
 		%{_sysconfdir}/modprobe.d/10-unsupported-modules.conf
 fi
 %endif
 
-# upgrade from old locations
-if test -e %{_sysconfdir}/modprobe.conf.local; then
-	mv -f %{_sysconfdir}/modprobe.conf.local \
-		%{_sysconfdir}/modprobe.d/99-local.conf
-fi
+check_lib_move() {
+    local pkg f
+
+    rm -f "/run/%{name}-lib-move"
+
+    # 99-local.conf was dropped with the /lib move
+    [ -f %{_sysconfdir}/modprobe.d/99-local.conf ] || return 0
+
+    # check if 99-local.conf is owned by suse-module-tools
+    pkg=$(rpm -qf --qf "%{name}" %{_sysconfdir}/modprobe.d/99-local.conf 2>/dev/null)
```

Comment:
I wouldn't do this in a scriptlet

Reply:
What harm does it do? Can you tell me some other way to figure out if the file, if it exists, belongs to the package being updated?

The point is that I want to avoid activating the "rename `.conf.rpmsave` to `.conf` code in `%posttrans` in all cases, except if I'm updating from the version that still holds config files in `/etc` to a version that has moved them to `/lib`. I think that's important.

Reply:
Figured it out ... once the `/etc/modprobe.d` files have been dropped from the package, no more `.rpmsave` files will be generated for them. Thus my `rpm -qf --qf "%{name}` check is redundant.

I resolved this in the version just pushed.